packages/angular_devkit/build_angular/src/extract-i18n/index.ts

```diff
@@ -237,6 +237,10 @@ export async function execute(
             /\.(css|scss|sass|styl|less)$/,
             path.join(__dirname, 'empty-export-default.js'),
           ),
+          new webpack.NormalModuleReplacementPlugin(
+            /^angular-resource:\/\/text\/(x-less|x-sass|x-scss|css)/,
```

Comment:
Only styles use this syntax so you can actually ignore all `angular-resource` URIs.  This removes the need to modify more than one package.

Reply:
Ok, I wasn't sure if it might be used for anything else so I figured it would be safer to pass the mime type, but I'll adjust it.